.vib/metrics-server/vib-publish.json

```diff
@@ -16,6 +16,42 @@
         }
       ]
     },
+    "verify": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/metrics-server"
+        },
+        "runtime_parameters": "c2VydmljZUFjY291bnQ6CiAgY3JlYXRlOiB0cnVlCiAgYXV0b21vdW50U2VydmljZUFjY291bnRUb2tlbjogdHJ1ZQpleHRyYUFyZ3M6Ci0gLS1hdXRob3JpemF0aW9uLWFsd2F5cy1hbGxvdy1wYXRocz0vbWV0cmljcwotIC0ta3ViZWxldC1pbnNlY3VyZS10bHM9dHJ1ZQpjb250YWluZXJQb3J0czoKICBodHRwczogODQ0NApzZXJ2aWNlOgogIHR5cGU6IExvYWRCYWxhbmNlcgogIHBvcnRzOgogICAgaHR0cHM6IDQ0Mwpjb250YWluZXJTZWN1cml0eUNvbnRleHQ6CiAgZW5hYmxlZDogdHJ1ZQogIHJ1bkFzVXNlcjogMTAwMg==",
```

Comment:
You are missing the `rbac.create=true` param here, which I believe is important for the chart to work properly

Reply:
You are right, changed now!